### PR TITLE
ci: add release process

### PR DESCRIPTION
**Pull request by beeme1mr**
Signed-off-by: Michael Beemer <beeme1mr@users.noreply.github.com>

### CODE REVIEW

File: .gherkin-lintrc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "scenario-size": [
+    "on",
+    {
+      "steps-length": {
+        "Background": 3,
+        "Scenario": 15
+      }
+    }
+  ]
+}
```
Comment (toddbaert) [inline .gherkin-lintrc:55]: Where did you find this?

File: .github/workflows/release-please.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+      release_tag_name: ${{ steps.release.outputs.tag_name }}
+
+  push-module:
+    needs: release-please
+    name: push schema to buf registry
```
Comment (toddbaert) [inline .github/workflows/release-please.yml:36]: I think this name is wrong.

**Review by skyerus — APPROVED** [review]

File: .gherkin-lintrc (hunk 1 above)
Comment (beeme1mr) [reply .gherkin-lintrc:55]: I found it on [this blog](https://medium.com/@virtim/how-we-implemented-gherkin-lint-and-why-you-should-too-6faac532d2a). The config worked OOTB with minor configuration changes.

File: .github/workflows/release-please.yml (hunk 2 above)
Comment (beeme1mr) [reply .github/workflows/release-please.yml]: Good catch, fixed!

**Pull request merged**